e2e/specs/return-to-patient-list.spec.ts

```diff
@@ -0,0 +1,177 @@
+import { test } from '../core';
+import { HomePage, PatientListsPage } from '../pages';
+import { expect } from '@playwright/test';
+import {
+  type Cohort,
+  type CohortMember,
+  type Patient,
+  addPatientToCohort,
+  deleteCohort,
+  deletePatient,
+  generateRandomCohort,
+  generateRandomPatient,
+  removePatientFromCohort,
+} from '../commands';
+
+let cohortMemberShip: CohortMember;
+let cohort: Cohort;
+let patient: Patient;
+
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+  cohort = await generateRandomCohort(api);
+  cohortMemberShip = await addPatientToCohort(api, cohort.uuid, patient.uuid);
+});
+
+test('Return to patient list from the patient chart', async ({ page }) => {
+  const homePage = new HomePage(page);
```

Comment:
This is not needed.

Reply:
The homePage variable is used when asserting the home page url: https://github.com/openmrs/openmrs-esm-patient-management/pull/957/files#diff-57388e0d22a908a325da8e63cb46dd7c304203cbbcc63761cc649c15c03491dcR38

Reply:
You can use `await expect(page).toHaveURL`